### ISSUE TITLE
Issue with python

### ISSUE THREAD

**Issue by michK** [wee-slack/wee-slack#738]
Good day,

I installed weechat with `brew install weechat`, and then when running weechat I get the error `ModuleNotFoundError: No module named 'websocket'`.

So, as the readme suggests, I tried to install websockets with `sudo /usr/local/opt/python@2/bin/pip2 install websocket_client`, but the problem is that I don't have a pip in my `/usr/local/opt/python@2/bin/` directory, or in `/usr/local/opt/python@3/bin/` for that matter.

I then installed a system pip with `sudo easy_install pip`, which installs pip to `/usr/local/bin/pip`. I then did `sudo pip install websocket_client`, which successfully installed, but I still get `ModuleNotFoundError` when I run weechat.

I then uninstalled the homebrew version of weechat, and built weechat from source, which seemed to work, but for some reason it didn't build with the python plugin.

Any ideas?
Thanks!

System: macOS Mojave, Version 10.14.5 (18F132)

**Comment by michK** [comment]
In case this is useful to someone, I solved this issue with the following:

- First I installed websocket-client manually by downloading from gere and then doing `sudo /usr/local/opt/python@2/bin/python setup.py install` from within the downloaded `websocket-client/` directory.
- This didn't make the problem go away, so I thought I'd check if homebrew didn't use python 3, which it turns out it did, so I did `sudo /usr/local/opt/python@3/bin/python3 setup.py install` from within `websocket-client/`

This fixed it for me, so it seems that homebrew now used python 3 by default.

Cheers!
